source/_components/media_player.blackbird.markdown

```diff
@@ -0,0 +1,64 @@
+---
+layout: page
+title: "Blackbird 8x8 HDMI Matrix Switch"
+description: "Instructions on how to integrate Monoprice Blackbird 4k 8x8 HDBaseT Matrix Switch into Home Assistant."
+date: 2018-03-23 16:45
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: monoprice.svg
+ha_category: Media Player
+ha_release: 0.66
+ha_iot_class: "Local Polling"
+---
+
+The `blackbird` platform allows you to control [Monoprice Blackbird Matrix Switch](https://www.monoprice.com/product?p_id=21819) using a serial connection.
+
+To add a Blackbird device to your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+media_player:
+  - platform: blackbird
+    port: /dev/ttyUSB0
+    zones:
+      1:
+        name: Main Bedroom
+      2:
+        name: Living Room
+      3:
+        name: Kitchen
+      4:
+        name: Bathroom
+      5:
+        name: Dining Room
+      6:
+        name: Guest Bedroom
+      7:
+        name: Outside
+      8:
+        name: Theater
+    sources:
+      1: 
+        name: BluRay
+      3:
+        name: Chromecast
+      8:
+        name: AppleTV
+```
+
+Configuration variables:
```

Comment:
Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration